aries-site/src/pages/index.js

```diff
@@ -13,8 +25,41 @@ const Index = () => {
     }
   }, []);
 
-  // return <Landing />;
-  return <About />;
+  return (
+    <PageLayout title="Home" isLanding>
+      {/* <Grid columns={['flex', 'xxlarge', 'flex']}> */}
+      {/* <Box /> */}
+      <Box gap="large">
+        <HomeTiles>
+          <Tile background="white">
+            <Image src="/image-hands.png" />
+          </Tile>
+          <IntroTile background="white" />
+        </HomeTiles>
+        <HomeTiles>
+          {data.map(item => (
+            <Tile
+              pad="medium"
```

Comment:
Thinking this should be bumped up to 'large' or something bigger than medium when the screen size = 'small'. Since the padding of "medium" is responsive and changes to a smaller value for small screens, the tile content feels a little bit too close to the edge when we get to mobile.